dmn-kotlin-dsl/src/main/kotlin/org/powerflows/kotlin/dsl/api.kt

```diff
@@ -0,0 +1,343 @@
+package org.powerflows.kotlin.dsl
```

Comment:
I thought about the package name. Imho the package should be renamed to org.powerflows.dmn.kotlin.dsl because this is strongly related to dmn project.

Reply:
ok